src/Helpers/async-pipe.test.ts

```diff
@@ -31,4 +32,13 @@ describe('asyncPipe', async (assert: any) => {
       expected: await incP(double(inc(await doubleP(value)))),
     })
   }
+
+  {
+    assert({
+      given: 'no starting value',
+      should,
+      actual: await asyncPipe(fixedNum, incP)().catch(identity),
```

Comment:
Is the catch here needed?

Reply:
Not really for this particular test, we do need a test for it though to make sure the types don't complain, I will remove the catch from all the current tests and make a specific test.